src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/Domain/TournamentsController.cs

```diff
@@ -768,6 +769,46 @@ private void FillRoundNames(ScheduleViewModel scheduleViewModel)
             scheduleViewModel.RoundNames = roundNames;
         }
 
+        /// <summary>
+        /// Fills home/away teams names with "Winner#" if they participate not in 
+        /// first round or bronze game.
+        /// </summary>
+        /// <param name="scheduleViewModel"></param>
+        private void FillPlayoffGamesTeamsNamesWithWinners(ScheduleViewModel scheduleViewModel)
+        {
+            byte winnerCounter = 1;
+            string winnerText = "Winner";
+            bool mustDisplayOnlyTeamNames;
+
+            foreach (var pair in scheduleViewModel.Rounds)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add curly braces around the nested statement(s) in this 'foreach' block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS121)